src/angular-selectbox.js

```diff
@@ -108,6 +108,12 @@
 
       $scope.$on('$destroy', _unbind);
 
+      $scope.$watch('vm.value',function(){
+          // watches value changes from external locations.
+          // Useful for working with socket.io
+          _getSelected();
```

Comment:
@milica, thank you for your job!
I have found similar bug in my apps, and that solution resolve my problem.
@mertpamukcu, great that you found it;
But we have one more problem;
We need to delete additional call 
`_getSelected();` 
From function 'change' at line 177, because onclick this function have additional call;